apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -167,42 +167,55 @@ function CustomEventDialogs(props: CustomEventDialogProps) {
                     </IconButton>
                 </Tooltip>
             )}
-            <Dialog open={open} onClose={handleClose} maxWidth={'lg'}>
+            <Dialog open={open} onClose={handleClose} maxWidth={'xs'}>
+                <DialogTitle id="form-dialog-title">
+                    {props.customEvent ? 'Edit a Custom Event' : 'Add a Custom Event'}
+                </DialogTitle>
                 <DialogContent>
-                    <FormControl>
-                        <InputLabel htmlFor="EventNameInput">Event Name</InputLabel>
-                        <Input required={true} value={title} onChange={handleEventNameChange} />
+                    <FormControl fullWidth>
+                        <TextField
+                            id="event-name-input"
+                            label="Event Name"
+                            variant="outlined"
+                            required={true}
+                            value={title}
+                            onChange={handleEventNameChange}
+                            margin="dense"
+                        />
                     </FormControl>
-                    <form noValidate style={{ display: 'flex', gap: 5, marginTop: 5 }}>
+                    <FormControl
+                        fullWidth
+                        sx={{ display: 'flex', flexDirection: 'row', gap: '12px', marginTop: '12px' }}
```

Comment:
```suggestion
                        sx={{ display: 'flex', flexDirection: 'row', gap: '12px' }}
```
repeat: margin is a bad pattern for maintainability